### DISCUSSION TITLE
Is it possible to use hexadecimal colors in 'event-color' prop. QDate component Quasar V2

### DISCUSSION THREAD

**Discussion by lucasdavidferrero** [quasarframework/quasar#12298] [General - Components / Directives / etc]
Hi. I've a requirement that implies using different hexadecimal colors for each event in QDate.

Using 'event-color' prop doesn't solve this, since it only accepts Quasar pre-defined colors. 

Is there a way to apply a hexadecimal color to an event in QDate component?

**Comment by MilosPaunovic** [comment]
You can add your own custom colors and use 'em in `QDate`:
[https://quasar.dev/style/color-palette#adding-your-own-colors](https://quasar.dev/style/color-palette#adding-your-own-colors)

**Reply by lucasdavidferrero** [reply]
This hexadecimal value comes from a database

**Comment by MilosPaunovic** [comment]
Might not be the most elegant solution, but it will get you what you need:
[https://codepen.io/milospaunovic/pen/rNYeYOb?editors=1111](https://codepen.io/milospaunovic/pen/rNYeYOb?editors=1111)
